web/themes/custom/westy/src/scss/components/_back-to-top.scss

```diff
@@ -0,0 +1,7 @@
+#back-to-top {
+    position: fixed;
+    bottom: 4rem;
+    right: 9rem;
+    z-index: $zindex-popover;
+  }
+  
```

Comment:
Needs a newline per Github.

Reply:
file updated with new line

Reply:
This is odd. It is still showing the need for a newline.

![2022-08-26 at 4 59 PM](https://user-images.githubusercontent.com/4097978/187005442-8585b4c0-b674-4fbc-983c-c953cd0b76d2.png)

@kxwang , @jookyg , or @RichardDavies have you seen this before?